suite-native/trading-atoms/src/components/TradeInfo/ProviderWaitingCard.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from 'react';
+import { FadeIn, FadeOut } from 'react-native-reanimated';
+
+import { AnimatedBox, Spinner, Text, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+
+export type ProviderWaitingCardProps = {
+    title: ReactNode;
+};
+
+export const ProviderWaitingCard = ({ title }: ProviderWaitingCardProps) => (
```

Comment:
Q: It is half way through being universal component so meybe we can make it really universal. Wdyt?

```suggestion
export type WaitingCardProps = {
    title: ReactNode;
    subtitle: ReactNode;
};


export const WaitingCard = ({ title, subtitle }: WaitingCardProps) => (
```

No hard opinion on this